frontends/election-manager/src/components/smartcard_modal/card_details_view.tsx

```diff
@@ -94,87 +98,81 @@ export function CardDetailsView({
     });
   }
 
-  function openUnprogramCardConfirmationModal() {
-    setIsUnprogramCardConfirmationModalOpen(true);
+  async function unprogramCard() {
+    setActionStatus({
+      action: 'Unprogram',
+      role,
+      status: 'InProgress',
+    });
+    const result = await card.unprogramUser();
+    setActionStatus({
+      action: 'Unprogram',
+      role,
+      status: result.isOk() ? 'Success' : 'Error',
+    });
   }
 
-  function closeUnprogramCardConfirmationModal() {
-    setIsUnprogramCardConfirmationModalOpen(false);
-  }
+  const electionDisplayString = doesCardElectionHashMatchMachineElectionHash
+    ? electionToDisplayString(electionDefinition.election)
+    : 'Unknown Election';
 
-  let electionDisplayString = 'Unknown';
-  if (doesCardElectionHashMatchMachineElectionHash) {
-    const { election } = electionDefinition;
-    const electionDateFormatted = format.localeWeekdayAndDate(
-      new Date(election.date)
-    );
-    electionDisplayString = `${election.title} — ${electionDateFormatted}`;
-  } else if (role === 'superadmin') {
-    electionDisplayString = 'N/A';
+  const possibleActions = new Set<SmartcardActionStatus['action']>();
+  if (
+    'passcode' in programmedUser &&
+    (role === 'superadmin' ||
+      // If the card is from a prior election, no need to display PIN resetting. Unprogramming is
+      // the only meaningful action in this case
+      doesCardElectionHashMatchMachineElectionHash)
+  ) {
+    possibleActions.add('PinReset');
+  }
+  // Don't allow unprogramming super admin cards to ensure election officials don't get
+  // accidentally locked out. Likewise prevent unprogramming when there's no election definition on
+  // the machine since cards can't be programmed in this state
+  if ((role === 'admin' || role === 'pollworker') && electionDefinition) {
+    possibleActions.add('Unprogram');
   }
 
   return (
-    <Prose textCenter>
-      <h2>Card Details</h2>
-      {/* An empty div to maintain space between the header and subsequent p. TODO: Consider adding
-        a `maintainSpaceBelowHeaders` prop to `Prose` */}
-      <div />
+    <Prose textCenter theme={fontSizeTheme.medium}>
```

Comment:
Why was this changed from large?

Reply:
Large was spilling outside the bounds of my 14-inch MacBook screen, but I may not be testing on the right resolution. What's the resolution of the ThinkPads that we run VxAdmin on?

Reply:
^ Got the resolution of the ThinkPads but also collectively decided with @beausmith that medium text is fine. What's really most important is that the PIN text is larger relative to other text, which it still is at `1.5em`